### ISSUE TITLE
Images

### ISSUE THREAD

**Issue by ChickCoder123** [thelegendski/ski.js#35]
Images cannot render using thumbnail.js because of the cross-origin property. I think adding img.crossOrigin = 'anonymous' will  fix this.
![Screenshot 2023-08-10 3 30 57 PM](https://github.com/thelegendski/ski.js/assets/129548862/344e8045-eb62-4759-832e-0db4ceca28b3)

**Comment by ChickCoder123** [comment]
I have checked that my image is allowed and the element is above the script tag. I have no idea why your image() function is not responding to it. I tried ctx.drawImage with my image and it worked completely fine. Can you show me your full code for the image function.

Also, have you considered coming back to KA?

**Comment by thelegendski** [comment]
i understand the element is above the script tag. as soon as you press the escape key, **thumbnail.js** deletes its very existence from the webpage so any reference is void.
bro, it's open-source code, :p. `image` doesn't work the same way. you're lookin' for `loadImage` which would require a demo. hence, i asked for the image `src`, :). https://github.com/thelegendski/ski.js/blob/297230a8d724c8f9777eddd4124739de9feccb8a/main.js#L1436
`ctx.drawImage` worked? you do realize that CORS kicks in prior to `ctx.drawImage`, o.O? `image` is pretty much a wrapper for `ctx.drawImage`, so idk how that's possible if thumbnail.js already deleted the image.

yes. i'll send a ban appeal in november as the Guardians were pretty fed up.

**Comment by ChickCoder123** [comment]
Oh, okay. I understand.

Yeah, I know PJS front to back, top to bottom. I used
`var img = loadImage('myImageSourceFromWikipedia')`

Yeah, it worked when I did document.createElement after realizing that thumbnail.js kept on returning null on elements outside of script. I didn't need to configure with cross origin at all for some reason. 🤨

Okay. Wonder how people will react when you return. Warning: The KACP is basically dead other than new people who ask for votes and brag about knowing how to use ellipse()
